components/vf-summary/vf-summary--profile.scss

```diff
@@ -30,7 +30,7 @@
   @include inline-link;
 }
 .vf-summary__link--secondary {
-  @include inline-link($vf-link---secondary-color, $vf-link---secondary-color--hover, $vf-link---secondary-color--visited, $vf-include-normalisations: true);
+  @include inline-link(set-color(vf-color--grey--darkest), set-ui-color(vf-ui-color--black), set-color(vf-color--purple), $vf-include-normalisations: true);
```

Comment:
How much is left to get this PR ready? It'd be really nice to get in as this CSS will fix the issue around white on white links.

Reply:
Or, is what is already done usable, and we can move the remaining work to a new issue?